content/en/guide/v10/typescript.md

```diff
@@ -350,15 +358,13 @@ const Counter = ({ initial = 0 }) => {
 
 `useEffect` does extra checks so you only return cleanup functions.
 
-```typescript
+```ts
 useEffect(() => {
   const handler = () => {
     document.title = window.innerWidth.toString();
   };
   window.addEventListener("resize", handler);
 
-  // ✅  if you return something from the effect callback
-  // it HAS to be a function without arguments
```

Comment:
It seems this line is mere explanation about `useEffect`. 
No one passes an argument since this function does not accept arguments.
Therefore, this lines is unnecessary.